lib/iris/tests/results/nimrod/load_2flds.cml

```diff
@@ -1,41 +1,38 @@
 <?xml version="1.0" ?>
 <cubes xmlns="urn:x-iris:cubeml-0.2">
-  <cube dtype="int16" long_name="Visibility" units="unknown">
+  <cube dtype="float32" long_name="Visibility" units="m">
     <attributes>
       <attribute name="field_code" value="155"/>
-      <attribute name="invalid_units" value="m/2-25k"/>
+      <attribute name="institution" value="Met Office"/>
       <attribute name="nimrod_version" value="2"/>
       <attribute name="num_model_levels" value="1"/>
       <attribute name="source" value="UM?"/>
+      <attribute name="title" value="Unknown"/>
     </attributes>
     <coords>
-      <coord>
-        <dimCoord id="9c8bdf81" points="[355014.0]" shape="(1,)" standard_name="forecast_reference_time" units="Unit('hours since 1970-01-01 00:00:00', calendar='gregorian')" value_type="float64"/>
+      <coord datadims="[0]">
+        <dimCoord id="b40ecfd3" points="[10800, 31546800]" shape="(2,)" standard_name="forecast_period" units="Unit('second')" value_type="int32"/>
```

Comment:
I'm noticing that these time coords have changed from floats to ints, which seems a bit unusual.
Is this really correct ?
Same is visible in a couple more -- I'll flag those separately ...

Reply:
This is the IMPROVER standard. It is much more precise to cast time as an integer number of seconds, otherwise you start losing fractions of time, even at float64, when you manipulate them. IMPROVER uses int64 for time and forecast_reference_time (seconds since 1970) and int32 for forecast_period (seconds).

Reply:
Ok!